src/Page/Food.elm

```diff
@@ -294,6 +308,43 @@ update ({ queries } as session) msg model =
             , Cmd.none
             )
 
+        ProductsLoaded (RemoteData.Success products) ->
+            let
+                initialQuery =
+                    if model.initialQuery /= Query.emptyQuery then
+                        model.initialQuery
+
+                    else
+                        -- If we didn't load a specific query, then default to the carrot cake... if we've loaded it!
+                        products
+                            |> List.filter (\product -> product.name == "Carrot cake (643g)")
```

Comment:
How about picking the first product in the list instead, ensuring it's always the carrot cake? Or alternatively, adding a `priority` field to have the desired default product identified as such? Relying on the label seems  unnecessarily prone to desyncing.